python-checks/src/main/java/org/sonar/python/checks/FlaskHardCodedSecret.java

```diff
@@ -117,40 +115,40 @@ private static boolean isCallToDictConstructor(CallExpression callExpression) {
       .isPresent();
   }
 
-  private static boolean hasIllegalKeyValuePair(DictionaryLiteral dictionaryLiteral) {
+  private boolean hasIllegalKeyValuePair(DictionaryLiteral dictionaryLiteral) {
     return dictionaryLiteral.elements().stream()
       .filter(KeyValuePair.class::isInstance)
       .map(KeyValuePair.class::cast)
       .map(KeyValuePair::key)
```

Comment:
need to validate `KeyValuePair`'s value as well that it is string literal
Sample test case is:
```python
def foo(x):
  app.config.update({"SECRET_KEY": x}) # FP here
```

Reply:
Thanks for the tip. I improved this in d194803430d05b7ae392a3a54adb07f5d392cbd5.